company_supplier_context/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Company Supplier Context",
+    "summary": """
+        Keep only companies in supplier context""",
```

Comment:
I don't think this description is correct, but I may be saying this in ignorance. I think the correct description is:

> When creating a new supplier, make it a company partner type by default.

I don't think that the functionality filters out non-company suppliers.

Reply:
Indeed, fixing it 👌